### ISSUE TITLE
Should the error message be updated to `givien` instead of `implicit` when the given instance of missing

### ISSUE THREAD

**Issue by hepin1989** [lampepfl/dotty#14717]
## Compiler version

Scala3.1.1

## Minimized code

```Scala
summon[Int]
```

## Output

```scala
no implicit argument of type Int was found for parameter x of method summon in object Predef
```

## Expectation
`no given instance of type Int was found for parameter x of method summon in object Predef`

As in Scala 3 we are providing an using argument which is a `given` instance.

**Comment by ckipp01** [comment]
Thanks for submission! I just push #14718 to address this, but it can also probably be marked as a duplicate of #7710.

**Comment by odersky** [comment]
I just merged #14718. @ckipp01 can this issue be closed now, or should we leave it open?

**Comment by ckipp01** [comment]
Yup, this one is now taken care of with the changes in #14718 
```
scala> summon[Int]
-- Error: ----------------------------------------------------------------------
1 |summon[Int]
  |           ^
  |no given instance of type Int was found for parameter x of method summon in object Predef
1 error found
```
